qan-api2/services/analytics/profile.go

```diff
@@ -28,18 +28,18 @@ import (
 // GetReport implements rpc to get report for given filtering.
 func (s *Service) GetReport(ctx context.Context, in *qanpb.GetReportRequest) (*qanpb.GetReportResponse, error) {
 	if in.PeriodStartFrom == nil || in.PeriodStartTo == nil {
-		return nil, fmt.Errorf("from-date: %s or to-date: %s cannot be empty", in.PeriodStartFrom, in.PeriodStartTo)
+		return nil, fmt.Errorf("from-date: %v or to-date: %v cannot be empty", in.PeriodStartFrom, in.PeriodStartTo)
 	}
 
 	periodStartFromSec := in.PeriodStartFrom.Seconds
 	periodStartToSec := in.PeriodStartTo.Seconds
 	if periodStartFromSec > periodStartToSec {
-		return nil, fmt.Errorf("from-date %s cannot be later then to-date %s", in.PeriodStartFrom, in.PeriodStartTo)
+		return nil, fmt.Errorf("from-date %v cannot be later then to-date %v", in.PeriodStartFrom, in.PeriodStartTo)
 	}
 	periodDurationSec := periodStartToSec - periodStartFromSec
 
 	if _, ok := standartDimensions[in.GroupBy]; !ok {
-		return nil, fmt.Errorf("unknown group dimension: %s", in.GroupBy)
+		return nil, fmt.Errorf("unknown group dimension: %q", in.GroupBy)
```

Comment:
`%q` is quite hard to query for in the logs because of escaping, I'd rather leave it as is.

Reply:
I updated format to %#q so it will be quoted, but not escaped.